Source/Logger.swift

```diff
@@ -0,0 +1,68 @@
+import Venice
+public class Logger {
```

Comment:
Since we generally don't use inheritance in Zewo (very few minor exceptions), its probably best to continue that pattern and mark this class with `final`. Also, this is OCD but I like a line break after the import statements.